src/wp-admin/includes/plugin.php

```diff
@@ -892,6 +892,10 @@ function activate_plugins( $plugins, $redirect = '', $network_wide = false, $sil
 function delete_plugins( $plugins, $deprecated = '' ) {
 	global $wp_filesystem;
 
+	if ( '' !== $deprecated ) {
+		_deprecated_argument( __FUNCTION__, '2.6.0' );
```

Comment:
```suggestion
		_deprecated_argument( __FUNCTION__, '4.0.0' );
```
Appears to be deprecated in https://core.trac.wordpress.org/ticket/28111.